README.md

```diff
@@ -10,6 +10,35 @@
 
 The technical documentation is available [here](https://ivadomed.org).
 
+## Installation
+
+``ivadomed`` requires Python >= 3.6 and PyTorch >= 1.5.0. We recommend working under a virtual environment, which could be set as follows:
```

Comment:
do we want to keep the py>3.6 if we know it fails with py3.9?

Reply:
I was thinking we could fix this once we investigated #556 --> Future PR.